### PR TITLE
added embedding of merchant map

### PR DESCRIPTION
**Pull request by IfaqYurmama**
embedding http://bit.ly/nav-accepted-here

### CODE REVIEW

File: content/merchants/index.md [hunk 1]
```diff
@@ -19,7 +19,17 @@ type: "default"
 <p>Anyone can accept NavCoin on their website or in their store to receive instant global payments.</p>
 <p>If your store accepts NavCoin and you would like to be listed on this page, read about how you can contribute to this website.
 {{< /left_image_section >}}
-
+    
+{{< home_centre_text_section
+    title="World map of all stores and merchants accepting NAV"
```
Comment (craigmacgregor) [inline content/merchants/index.md]: I'd probably remove the word "all" here because it's not a complete list.

File: content/merchants/index.md [hunk 2]
```diff
@@ -19,7 +19,17 @@ type: "default"
...
+    linkBgColor="#43799a"
+    gradient=""
+    fontColor="#FFF">}}
+    <div>
+      <iframe src="https://www.google.com/maps/d/embed?mid=1wzMJAyujqSQ-Iyy0mqQxP_uuwrphIuYJ&hl=en" width="960" height="720"></iframe>
+    </div>
+    {{< /home_centre_text_section>}}
+    
 {{< links_section
```
Comment (craigmacgregor) [inline content/merchants/index.md]: We should also add some instruction of how to add your store to the map via the google form (or talking to mods on discord) and link to the form.

**Comment by matt-auckland** [comment]
image

Looks fine on a desktop but on smaller screens the map is too big

File: content/merchants/index.md [hunk 3]
```diff
@@ -20,6 +20,19 @@ type: "default"
 <p>If your store accepts NavCoin and you would like to be listed on this page, read about how you can contribute to this website.
 {{< /left_image_section >}}
 
+{{< home_centre_text_section
+    title="World map of stores and merchants accepting NAV"
+    linkBgColor="#43799a"
+    gradient=""
+    fontColor="#FFF">}}
+    <div class="merchant-map">
+      <iframe src="https://www.google.com/maps/d/embed?mid=1wzMJAyujqSQ-Iyy0mqQxP_uuwrphIuYJ&hl=en" width="720" height="540"></iframe>
```
Comment (ro-savage) [inline content/merchants/index.md]: You should be able to remove the `width` and `height` here and do it in the style and percentages and `max-width` / `max-height` to allow working on mobile. (Untested)

**Comment by IfaqYurmama** [comment]
@ro-savage @matt-auckland I think I have solved this issue, can you check if it works?

**Comment by craigmacgregor** [comment]
@IfaqYurmama can you put a bit of spacing between the text underneath and the map? Perhaps:
```
.add-merchant { margin-top: 20px; }
```

**Comment by craigmacgregor** [comment]
There's also a white gap between the blue horizontal section and the grey map section. I think this is just caused by the line break in the .md file. Try removing the empty line between these shortcodes and see if the empty <p></p> tag goes away.

screen shot 2018-05-03 at 7 35 50 am

File: content/merchants/index.md [hunk 4]
```diff
@@ -20,6 +20,20 @@ type: "default"
 <p>If your store accepts NavCoin and you would like to be listed on this page, read about how you can contribute to this website.
 {{< /left_image_section >}}
```
Comment (craigmacgregor) [inline content/merchants/index.md]: try removing this empty line to fix the empty ```<p>``` tag causing that white space between sections

**Comment by IfaqYurmama** [comment]
Please excuse that mess of commits but I think I fixed your notes @craigmacgregor

**Comment by matt-auckland** [comment]
@IfaqYurmama you missed the whitespace in the merchants page index.md, but I've removed it for you

**Comment by ro-savage** [comment]
I am closing this PR as no follow up work has taken place. Feel free to re-open if something changes.

**Pull request closed without merging**